### PR TITLE
Add support for the zonal case and go back to the inertial reference frame

### PR DESCRIPTION
**Pull request by pleroy**
Benchmark when using the new code in the Ephemeris in Release mode (something that this CL doesn't do).  Before:
```
Run on (4 X 3310 MHz CPU s)
10/01/18 21:17:14
----------------------------------------------------------------------------------------
Benchmark                                                 Time           CPU Iterations
----------------------------------------------------------------------------------------
BM_EphemerisSolarSystemMajorBodiesOnly/-3        45019792771 ns 44943888100 ns          1 +9.92179030813603147e-01 ua
BM_EphemerisSolarSystemMinorAndMajorBodies/-3    109432618748 ns 108779497300 ns          1 +9.92179031500272979e-01 ua
BM_EphemerisSolarSystemAllBodiesAndOblateness/-3 570587537435 ns 565612825700 ns          1 +9.92178864992771969e-01 ua
```
After:
```
Run on (4 X 3310 MHz CPU s)
10/02/18 23:43:21
----------------------------------------------------------------------------------------
Benchmark                                                 Time           CPU Iterations
----------------------------------------------------------------------------------------
BM_EphemerisSolarSystemMajorBodiesOnly/-3        46441258970 ns 45723893100 ns          1 +9.92179030813603147e-01 ua
BM_EphemerisSolarSystemMinorAndMajorBodies/-3    113307575563 ns 111337913700 ns          1 +9.92179031500272979e-01 ua
BM_EphemerisSolarSystemAllBodiesAndOblateness/-3 286661034049 ns 284608224400 ns          1 +9.92178865021134948e-01 ua
```

### CODE REVIEW

File: physics/oblate_body_body.hpp [hunk 1]
```diff
@@ -82,6 +84,19 @@ OblateBody<Frame>::Parameters::ReadFromMessage(
                           LegendreNormalizationFactor(2, 0) * reference_radius *
                           reference_radius;
 
+  // Zonalness.
+  parameters.is_zonal_ = true;
+  if (!(message.has_zonal() && message.zonal())) {
```
Comment (eggrobin) [inline physics/oblate_body_body.hpp]:
```C++
if (!message.zonal()))
```
given that the default value is `false`?

File: physics/rotating_body.hpp [hunk 2]
```diff
@@ -71,6 +71,12 @@ class RotatingBody : public MassiveBody {
   // at construction.
   Vector<double, Frame> const& polar_axis() const;
 
+  // Two unit vectors in the equatorial plane of the body.  |biequatorial| is
+  // also in the equatorial plane of Frame.  The basis |biequatorial|,
+  // |equatorial|, |polar_axis| is a direct one.
```
Comment (eggrobin) [inline physics/rotating_body.hpp]: is direct.

File: physics/oblate_body_body.hpp (hunk 1 above)
Comment (pleroy) [reply physics/oblate_body_body.hpp]: Done.

File: physics/rotating_body.hpp (hunk 2 above)
Comment (pleroy) [reply physics/rotating_body.hpp]: Done.

**Pull request merged**